numpy/lib/npyio.py

```diff
@@ -1573,8 +1593,9 @@ def _genfromtxt_dispatcher(fname, dtype=None, comments=None, delimiter=None,
                            names=None, excludelist=None, deletechars=None,
                            replace_space=None, autostrip=None, case_sensitive=None,
                            defaultfmt=None, unpack=None, usemask=None, loose=None,
-                           invalid_raise=None, max_rows=None, encoding=None, *,
-                           like=None):
+                           invalid_raise=None, max_rows=None, encoding=None,
+                           ndmin=None,
+                           *, like=None):
```

Comment:
```suggestion
                           invalid_raise=None, max_rows=None, encoding=None, *,
                           ndmin=None, like=None):
```
New optional arguments are usually made keyword-only these days.